dap_aria_mapping/notebooks/novelty/pipeline_openalex_novelty.py

```diff
@@ -0,0 +1,81 @@
+""" 
+Script/pipeline to calculate novelty scores for OpenAlex papers
```

Comment:
would suggest creating pipelines/novelty for this script rather than "notebooks"

Reply:
Good point, will change!

Reply:
On that note, I see you are using typer - any reason for using it over argparse? I have no preference, but if you are going to build CLIs, we've been doing it exclusively with the latter (plenty of examples throughout the pipeline folder).

Reply:
I haven't used argparse, but typer is super easy to use (looks like a bit easier than argparse in terms 
of length of boilerplate code)

Reply:
@ampudia19  https://towardsdatascience.com/typer-probably-the-simplest-to-use-python-command-line-interface-library-17abf1a5fd3e